drivers/qwiic/qwiic.mk

```diff
@@ -3,7 +3,7 @@ ifneq ($(strip $(QWIIC_ENABLE)),)
   OPT_DEFS += -DQWIIC_ENABLE
   SRC += qwiic.c
   ifeq ($(filter "i2c_master.c", $(SRC)),)
-    SRC += i2c_master.c
+    QUANTUM_LIB_SRC += i2c_master.c
```

Comment:
I think it would be better to use LIB_SRC here.

Reply:
And, I think there is no need to have `ifeq ($(filter "i2c_master.c", $(SRC)),)` here.

Reply:
Ah, correct, didn't catch that, thanks.

Also, what's the difference between `LIB_SRC` and `QUANTUM_LIB_SRC`, actually?

Reply:
The position on the command line when linking is slightly different.

https://github.com/qmk/qmk_firmware/blob/9f8c8c53a51ec6bb8acd55683b1d024558b071ab/build_keyboard.mk#L326-L349

Reply:
Hmm, I made a mistake.

In common_features.mk, we had to use QUANTUM_LIB_SRC.

Reply:
I'm sorry I made a wrong point.